src/jmcomic/jm_config.py

```
@@ -1,11 +1,47 @@
+class JmTaskContextFormatter(logging.Formatter):
+    """Format the stable correlation fields from a JM task context."""
+
+    @staticmethod
+    def _task_context_prefix(context) -> str:
+        if not isinstance(context, dict):
+            return ''
+
+        fields = []
+        task_id = context.get('task_id')
+        if task_id is None:
+            # 没有 task_id 时，默认不打印 task_context，保持日志简洁
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)